api/src/routes/api.ts

```diff
@@ -49,8 +49,9 @@ router.put("/:category/:job/ingest", function(req, res, next) {
 });
 
 router.put("/:category/:job", function(req, res, next) {
-    //res.send(JSON.stringify(getJobFromRequest(req).metadata.upda));
-    //res.send(JSON.stringify( { status: 'ok' } ));
+    let job = getJobFromRequest(req);
+    job.metadata.validate(job, req.params);
```

Comment:
This doesn't seem to be complete yet... the idea is that the route should use the validate() method to figure out whether the incoming data is valid or not. If it's invalid, it needs to return a 400 status with an error message, and otherwise, it needs to save the valid data to disk.